src/functions/template-tags/general.php

```diff
@@ -156,6 +156,49 @@ function tribe_get_event_label_plural_lowercase() {
 	);
 }
 
+/**
+ * Undocumented function
```

Comment:
Should this get another name or is Undocumented function, correct?

Reply:
🤦🏼